### PR TITLE
Update what-s-new-in-tls-ssl-schannel-ssp-overview.md

### PR DESCRIPTION
**Pull request by joshern-ms**
Worked with EE Wayne McIntyre and found that this is the behavior and needs to be updated for the public.

### CODE REVIEW

**Comment by illfated** [comment]
Very well, looks OK to me so far. Would you like to consider adjusting the added sentence to be a colored Note blob, similar in appearance to this previous one?

> [!NOTE]
> The client-side implementation of RFC 5077 was added in Windows 8.

(Depending on how you feel about this, I may have a suggestion for a slight grammar adjustment to your added sentence, as well.)

**Comment by eross-msft** [comment]
@joshern-ms I can work with you to merge this update, but as a Microsoft employee, please use the private repo, https://github.com/MicrosoftDocs/windowsserverdocs-pr in the future. You will have more options from the writing team by going through there.

Do you want the text to appear as a note or simply as a section of text?

**Comment by joshern-ms** [comment]
Hi Elizabeth,

Please elaborate on using the private repro. I went to the link and it says 404 error. I want to ensure I follow the correct process going forward. The text can appear as a note. See below. Thanks!
NOTE: If System Mapper is enabled by the client application and SendTrustedIssuers is configured, System Mapper will add CN=NT Authority to the issuers list.

Regards,

Jose Hernandez
T3 Support Engineer | Cloud & Infrastructure Solutions | Directory Services (On-Prem AD)
Customer Service & Support | Microsoft Corporation
Next OOF: May 11th -15th training, May 18th out of office
•+1 (469) 775-2763 | • jose.m.hernandez@microsoft.com<mailto:jose.m.hernandez@microsoft.com> | •Mon-Fri 12pm-8pm CT
Manager: Steve Stoker | • robert.stoker@microsoft.com<mailto:robert.stoker@microsoft.com>  | •+1 (469) 775-1004
You may also call +1 (800) 936-3100 and ask for the next available engineer.
[cid:image001.png@01D62AF5.29F75F00]

File: WindowsServerDocs/security/tls/what-s-new-in-tls-ssl-schannel-ssp-overview.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ In Windows Server 2012 and Windows 8, changes were made to the underlying authen
 
 -   Compatibility to previous versions of Windows operating systems is preserved.
 
+Also note that if System Mapper is enabled by the client application and you have configured SendTrustedIssuers that system mapper will add CN=NT Authority to the issuers list.
```
Comment (illfated) [inline WindowsServerDocs/security/tls/what-s-new-in-tls-ssl-schannel-ssp-overview.md:65]: Suggestion for using a MarkDown Note blob (colored) to highlight the note text:
```suggestion
> [!NOTE]
> If System Mapper is enabled by the client application and you have configured `SendTrustedIssuers`, that system mapper will add `CN=NT Authority` to the issuers list.

```
I have also added MarkDown back ticks around the phrases "SendTrustedIssuers" and "CN=NT Authority", both to keep Machine Translation from automatically translating part of the text and also to slightly emphasize the phrases as technical terms.

You are of course free to modify my suggestion before committing it to your PR branch.

**Comment by eross-msft** [comment]
@joshern-ms  Yes, the 404 error is normal, I see it too :) It just means that you have to sign in to your GitHub account before you can access the repo. The repo needs to make sure you have the right credentials...after that, you can use the instructions here: https://github.com/MicrosoftDocs/windowsserverdocs-pr/blob/master/Contributor-guide/github-browser-updates.md (if you want to edit, but don't want to use a code editor) or here: https://github.com/MicrosoftDocs/windowsserverdocs-pr/blob/master/Contributor-guide/edit-existing-using-github.md (to edit using Visual Studio Code). Hope this helps! Thank you for all of your effort so far, it's greatly appreciated!

**Comment by eross-msft** [comment]
I will also go ahead and accept this pull request, so you won't have to re-do it.

**Pull request merged**